examples/init/INCAR_methane.rlx

```diff
@@ -12,7 +12,7 @@ ISIF=2
 ISMEAR=0
 SIGMA=0.20000
 IBRION=-1
-NSW=10
+NSW=0
```

Comment:
Why setting NSW=0 in a relaxation INCAR? With NSW=0 the configuration will not be relaxed.